system-tests/__snapshots__/deprecated_spec.ts.js

```diff
@@ -164,7 +164,84 @@ exports['deprecated before:browser:launch args / no mutate return'] = `
 
 `
 
-exports['deprecated before:browser:launch args / concat return returns once per spec'] = `
+exports['deprecated before:browser:launch args / fails when adding unknown properties to launchOptions'] = `
```

Comment:
With this system-test snapshot update, we are updating behavior for before:browser:launch. Previously it would be called before each spec. Since we are only launching the browser once, this is changing to only being called once per full test fun in firefox and chrome. In electron, we will keep the same behavior.